packages/@aws-cdk/aws-amplify/lib/domain.ts

```diff
@@ -21,6 +21,20 @@ export interface DomainOptions {
    * @default - use `addSubDomain()` to add subdomains
    */
   readonly subDomains?: SubDomain[];
+
+  /**
+   * Automatically create subdomains for connected branches
+   *
+   * @default false
+   */
+  readonly domainEnableAutoSubDomain: boolean;
```

Comment:
since this has a default, it should be optional.

Reply:
Good point, changing it

tks